### DISCUSSION TITLE
Support for SrcAsPath

### DISCUSSION THREAD

**Discussion by jwbensley** [akvorado/akvorado#1227] [Q&A]
Disclaimer; I haven't looked into the technicalities of this so probably there is something obvious I am missing...

I have search through the discussions, issues and PRs, and found no mention about adding support for a `SrcAsPath` filter like the `DstAsPath` filter. Is this simply because nobody wants this or because there is some technical limitation which means it's not feasible?

**Comment by vincentbernat** [comment]
For the destination AS path, we know this is the correct one. If we were to add a source AS path, it would be the destination AS path if we were sending back the packet, not the real AS path used to receive the packet.

**Reply by jwbensley** [reply]
I see what you mean. thanks for clarifying.
